cuneiform/riscv64.patch

```diff
@@ -0,0 +1,23 @@
+diff --git PKGBUILD PKGBUILD
+index 7fe0ef1..87c9c3f 100644
+--- PKGBUILD
++++ PKGBUILD
+@@ -12,13 +12,16 @@ license=('BSD')
+ depends=('libmagick')
+ makedepends=('cmake')
+ source=(https://launchpad.net/cuneiform-linux/${pkgver%.*}/${pkgver%.*}/+download/$pkgname-linux-$pkgver.tar.bz2
+-	build-fix.patch)
++	build-fix.patch
++	build-fix2.patch)
```

Comment:
Did you submit the patch to upstream? The naming here isn't good enough I think...